route_generator/include/route_generator.h

```diff
@@ -68,6 +96,9 @@ class RouteGenerator
     bool start_active_route_cb(cav_srvs::StartActiveRouteRequest &req, cav_srvs::StartActiveRouteResponse &resp);
     bool abort_active_route_cb(cav_srvs::AbortActiveRouteRequest &req, cav_srvs::AbortActiveRouteResponse &resp);
 
+    // new service callbacks, will replace old service call later
```

Comment:
Well. I added this new call back but I did not think we are fully ready to just do the switch since it will break the current functionality. I think the service might still use the original callback and update to new call back at a later time.

Reply:
Hmm I feel this will happen often for the next few sprints. I think we need to determine how best to handle these braking changes. This feels like the same issue we had with the planning nodes not being connected. Lets talk with John and Mae.

Reply:
Cool. I will disconnect some old callback and logic so we can keep this feature branch clean.